.github/workflows/swift.yml

```
@@ -4,17 +4,21 @@ on:
   push:
     branches: [ main ]
   pull_request:
+    types: [opened, synchronize, ready_for_review]
     branches: [ main ]
+    workflow_dispatch:
```

actionlint (1.7.7)

unexpected key "workflow_dispatch" for "pull_request" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)